ci/script.sh

```diff
@@ -14,6 +14,11 @@ main() {
     cross test --target $TARGET
     cross test --target $TARGET --release
 
+    ruby --version
+    ls -la
+    which license_finder
```

Comment:
guess these 3 commands were a test and can be removed?

Reply:
hmm, that seems to be outdated but let me squash the code changes and remove that entry correctly.